javalib/src/main/scala/java/util/concurrent/Flow.scala

```diff
@@ -0,0 +1,30 @@
+// Ported from Scala.js commit: fb20d6f dated: 20 Jan 2023
```

Comment:
```suggestion
// Ported from Scala.js commit: fb20d6f dated: 2023-01-20
```
ISO 8601 format is preferable.